src/components/Menu.js

```diff
@@ -280,7 +289,7 @@ export default class Menu extends React.Component<MenuDefaultProps, MenuProps, v
       ) : null;
 
     const buttonLabel = (
-      <span>
+      <span data-test={dataTestValue}>
```

Comment:
Should the data-test attribute be on this `<span>` or on the < Dropdown.Toggle> component in the render() method? That would also include the little down-facing caret and the border of the button that triggers the menu. This way, the data-test info will only be available if the user clicks on the span itself.

Reply:
Yes, makes sense to have `data-test` on the `<Dropdown.Toggle>` component. Will make the changes.